python_modules/dagster-test/dagster_test/fixtures/docker_compose.py

```diff
@@ -105,7 +105,11 @@ def connect_container_to_network(container, network):
 
 
 def disconnect_container_from_network(container, network):
-    subprocess.check_call(["docker", "network", "disconnect", network, container])
+    # subprocess.run instead of subprocess.check_call so we don't fail when
+    # trying to disconnect a container from a network that it's not connected to
+    subprocess.run(  # pylint: disable=subprocess-run-check
+        ["docker", "network", "disconnect", network, container]
+    )
```

Comment:
can we get something emitted in the logs on non-zero exits to trace back to for problems like this failing on the first invocation?

Reply:
Added some logging - the Docker daemon already logs the failure but this will let us control the copy a bit closer to the source.